CHANGES/111.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed an issue where replacing the path would also replace the query and fragment.
```

Comment:
Please update the text. Something like "add `keep_query` and `keep_fragment` flags to <enumerate affected methods> for optional keeping query string and fragment in the resulting URL". The exact text is up to you but please keep in mind: it would be nice if a user could get a brief idea of the change without digging into PR content.

Reply:
Sure, it makes more sense.
Done